<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnicalcClientLib::DoNotTouchItInYourProgram::OutputWindow</class>
 <widget class="QWidget" name="UnicalcClientLib::DoNotTouchItInYourProgram::OutputWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Результат вычислений</string>
  </property>
  <property name="windowIcon">
   <iconset resource="logo.qrc">
    <normaloff>:/res/Images/iis_logo_small.png</normaloff>:/res/Images/iis_logo_small.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="waitingPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Ожидание получения результатов вычисления с сервера</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="maximum">
          <number>0</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="textVisible">
          <bool>false</bool>
         </property>
         <property name="invertedAppearance">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton1">
         <property name="text">
          <string>Отмена</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="errorPage">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>В ходе обработки модели возникала проблема:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="errorView"/>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton3">
         <property name="text">
          <string>Закрыть</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="resultPage">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>392</width>
            <height>210</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="checkBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>заменить текущую модель</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="closeButton2">
           <property name="text">
            <string>Закрыть</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton2</sender>
   <signal>clicked()</signal>
   <receiver>UnicalcClientLib::DoNotTouchItInYourProgram::OutputWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>496</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButton3</sender>
   <signal>clicked()</signal>
   <receiver>UnicalcClientLib::DoNotTouchItInYourProgram::OutputWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButton1</sender>
   <signal>clicked()</signal>
   <receiver>UnicalcClientLib::DoNotTouchItInYourProgram::OutputWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
